packages/replay-next/components/errors/UnexpectedErrorForm.module.css

```diff
@@ -0,0 +1,6 @@
+.Details {
+  font-style: italic;
+}
+
+.Boilerplate {
```

Comment:
:hand: dead style?

Reply:
Not really. The view references this style, there just happens to be no new rules in it. TBH I dunno how we prefer to handle this case. I guess I _could_ just return an empty `<div>` if that's "clearer" (seemed subjective)